fix hardcoded paths, rely on specified dependencies via $PATH and $EBROOT*
author: Kenneth Hoste (HPC-UGent)
--- SqueezeMeta-1.0.0/scripts/SqueezeMeta_conf_original.pl.orig	2019-11-26 11:48:35.838688703 +0100
+++ SqueezeMeta-1.0.0/scripts/SqueezeMeta_conf_original.pl	2019-11-26 11:52:57.253843644 +0100
@@ -80,25 +80,25 @@

 #-- External software

-$metabat_soft="$installpath/bin/metabat2";
-$maxbin_soft="$installpath/bin/MaxBin/run_MaxBin.pl";
-$spades_soft="$installpath/bin/SPAdes/spades.py";
-$barrnap_soft="$installpath/bin/barrnap";
-$rdpclassifier_soft="java -jar $installpath/bin/classifier.jar";
-$bowtie2_build_soft="$installpath/bin/bowtie2/bowtie2-build";
-$bowtie2_x_soft="$installpath/bin/bowtie2/bowtie2";
-$bwa_soft = "$installpath/bin/bwa";
-$minimap2_soft = "$installpath/bin/minimap2";
-$diamond_soft="$installpath/bin/diamond";
-$hmmer_soft="$installpath/bin/hmmer/hmmsearch";
-$megahit_soft="$installpath/bin/megahit/megahit";
-$prinseq_soft="$installpath/bin/prinseq-lite.pl";
-$prodigal_soft="$installpath/bin/prodigal";
-$cdhit_soft="$installpath/bin/cd-hit-est";
-$toamos_soft="$installpath/bin/AMOS/toAmos";
-$minimus2_soft="$installpath/bin/AMOS/minimus2";
-$checkm_soft="$installpath/bin/checkm";
-$minpath_soft="python $installpath/bin/MinPath1.4.py";
-$canu_soft="$installpath/bin/canu/canu";
-$trimmomatic_soft="java -jar $installpath/bin/trimmomatic-0.38.jar";
-$dastool_soft="$installpath/bin/DAS_Tool/DAS_Tool";
+$metabat_soft="metabat2";
+$maxbin_soft="$ENV{'EBROOTMAXBIN'}/run_MaxBin.pl";
+$spades_soft="spades.py";
+$barrnap_soft="barrnap";
+$rdpclassifier_soft="java -jar $ENV{'EBROOTRDPMINCLASSIFIER'}/dist/classifier.jar";
+$bowtie2_build_soft="bowtie2-build";
+$bowtie2_x_soft="bowtie2";
+$bwa_soft = "bwa";
+$minimap2_soft = "minimap2";
+$diamond_soft="diamond";
+$hmmer_soft="hmmsearch";
+$megahit_soft="megahit";
+$prinseq_soft="prinseq-lite.pl";
+$prodigal_soft="prodigal";
+$cdhit_soft="cd-hit-est";
+$toamos_soft="AMOS/toAmos";
+$minimus2_soft="AMOS/minimus2";
+$checkm_soft="checkm";
+$minpath_soft="MinPath$ENV{'EBVERSIONMINPATH'}.py";
+$canu_soft="canu";
++$trimmomatic_soft="java -jar $ENV{'EBROOTTRIMMOMATIC'}/trimmomatic-$ENV{'EBVERSIONTRIMMOMATIC'}.jar";
++$dastool_soft="DAS_Tool";
--- SqueezeMeta-1.0.0/utils/sqm_hmm_reads.pl.orig	2019-11-26 11:53:35.804013957 +0100
+++ SqueezeMeta-1.0.0/utils/sqm_hmm_reads.pl	2019-11-26 11:54:08.748159508 +0100
@@ -15,7 +15,7 @@

 use File::Basename;
 my $utilsdir = dirname(__FILE__);
-my $shortpair_soft = "$utilsdir/../bin/Short-Pair/Short-Pair.py";
+my $shortpair_soft = "Short-Pair.py";

 my($numthreads,$pair1,$pair2,$pfam,$outfile,$dietext,$tempfile1,$tempfile2,$hel);

